functions.py

```diff
@@ -1,84 +1,133 @@
+from command_factory import *
+from ICommand import ICommand
 import json
 import os
+import keys
 
 
-def search(path, primary_key):
-    y = False
-    for r, d, f in os.walk(path):
-        if primary_key in f:
-            y = True
-            break
-    return y
-
-
-def creates_dir(schema):
-    x = json.load(open(schema, 'r'))
-    p = os.path.join(os.getcwd(), x['database_name'])
-    os.mkdir(p)
-    for i in x['Tables']:
-        d = os.path.join(p, i['name'])
-        os.mkdir(d)
-
-
-def creates(schema, table, primary_key):
-    x = json.load(open(schema, 'r'))
-    path = os.getcwd() + '\\' + x['database_name'] + '\\' + table + '\\' + primary_key
-    f = open(path, 'w')
-    f.write("{\n")
-    i = 0
-    for k in x['Tables']:
-        if k['name'] != table:
-            i = i + 1
+
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self.schema = schema
+
+    def execute(self):
+        if self.schema is None:
+            return "Schema not found"
+        data = json.load(open(self.schema, 'r'))

```

Comment:
add this in constructor, we do I need to care about loading the schema here

Reply:
I don't understand this please explain why ?

Reply:
What is the idea about this, we made the constructor only gives the class variables their values, to be specific while every other function is run through the execute command.

Reply:
Now, when I read the execute code. Why do I care about schema if it is a string or file path ? 
Imagine someone later descided to add another function in command and wants to read schema, do he/she need to parse it again ? 

Why should I think about the schema format ?